src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
hmm, i think RGB alone makes sense. In essence we only care about scene-linear values. Ingested material (textures) are to be linearized, but as an operation that wouldn't necessarily be part of the renderer (it can be done with a tool such as maketx for example). And an output render are to be encoded depending on format chosen and standard chosen, also not necessarily part of the renderer (one could save scene-linear files only for example, and leave the conversion to an external tool). I'm not saying that should be done though, just that everywhere RGB is linear, and that linearization and encoding are pre and post render stages not necessarily part of the render, with very well defined contexts. 
For the sake of completion, the convention is usually to use for example, RGB for scene-linear, and R'G'B' for gamma encoded values. (RGB prime).